src/commands/add/addCredential.ts

```diff
@@ -21,13 +21,16 @@ import { getAndValidateServerUrl, getCredentialsInput } from './internal/input'
  */
 export const addCredential = async (
   targetName: string,
-  insecure: boolean = false
+  insecure: boolean | undefined = undefined
```

Comment:
Is this really necessary?

Reply:
Actually, no.
thanks for highlighting, I have double-check and updated.